docs/software/sciapps/cp2k.md

```diff
@@ -1,4 +1,400 @@
+                MEMORY 1200
+                NUMBER_PROC 1
+                &INTEGRALS
+                  &WFC_GPW
+                    CUTOFF 300
+                    EPS_FILTER 1.0E-12
+                    EPS_GRID 1.0E-8
+                    REL_CUTOFF 50
+                  &END WFC_GPW
+                &END INTEGRALS
+                &RI_MP2
+                &END RI_MP2
+              &END WF_CORRELATION
+              &XC_FUNCTIONAL NONE
+              &END XC_FUNCTIONAL
+            &END XC
+          &END DFT
+          &SUBSYS
+            &CELL
+              ABC 15.6404 15.6404 15.6404
+            &END CELL
+            &KIND H
+              BASIS_SET cc-TZ
+              BASIS_SET RI_AUX RI-cc-TZ
+              POTENTIAL GTH-HF-q1
+            &END KIND
+            &KIND O
+              BASIS_SET cc-TZ
+              BASIS_SET RI_AUX RI-cc-TZ
+              POTENTIAL GTH-HF-q6
+            &END KIND
+            &TOPOLOGY
+              COORD_FILE_FORMAT XYZ
+              COORD_FILE_NAME ./H2O-128.xyz
+            &END TOPOLOGY
+          &END SUBSYS
+        &END FORCE_EVAL
+        ```
+
+    All calculations run for this scaling tests were using 32 MPI ranks per node and 8 OpenMP threads per rank. 
+    The smallest amount of nodes necessary to run this calculation is 8.
+
+
+    | Number of nodes | Wall time (s) | Speecup | Efficiency |
+    |:---------------:|:-------------:|:-------:|:----------:|
+    | 8               |  2037.0       | 1.00    | 1.00       |
+    | 16              |  1096.2       | 1.85    | 0.92       |
+    | 32              |  611.5        | 3.33    | 0.83       |
+    | 64              |  410.5        | 4.96    | 0.62       |
+    | 128             |  290.9        | 7.00    | 0.43       |
+
+    MP2 calculations scale well on GH200, up to a large number of nodes ($> 50\%$ efficiency with 64 nodes).
+
+??? example "Scaling of `QS_mp2_rpa/128-H2O/H2O-128-RI-dRPA-TZ` benchmark"
+
+    The [`QS_mp2_rpa/128-H2O/H2O-128-RI-dRPA-T` benchmark](https://github.com/cp2k/cp2k/blob/master/benchmarks/QS_mp2_rpa/128-H2O/H2O-128-RI-dRPA-TZ.inp)
+    is a RPA energy calculation, traditionally used to benchmark the performance of the [COSMA] library. 
+    It a very large calculation, which requires at least 8 GH200 nodes to run. 
+    The calculations were run with 16 MPI ranks per node and 16 OpenMP threads per rank.
+    For RPA workloads, a higher ratio of threads per rank were beneficial.
+
+    | Number of nodes | Wall time (s) | Speecup | Efficiency |
+    |:---------------:|:-------------:|:-------:|:----------:|
+    | 8               |  575.4        | 1.00    | 1.00       |
+    | 16              |  465.8        | 1.23    | 0.61       |
+    | 32              |  281.1        | 2.04    | 0.51       |
+    | 64              |  205.3        | 2.80    | 0.35       |
+    | 128             |  185.8        | 3.09    | 0.19       |
+
+    This RPA input scales well until 32 GH200 nodes.
+
+### Known issues
```

Comment:
How about moving "known issues" to the bottom of the page.
I am leaning towards having FAQ and "common problems" etc style content at the bottom of pages across the whole documentation.

Reply:
Sounds good to me.